lib/beaker/dsl/ezbake_utils.rb

```diff
@@ -117,27 +116,90 @@ def install_ezbake_deps host
           end
 
         else
-          raise "No repository installation step for #{variant} yet..."
+          raise RuntimeError, "No repository installation step for #{variant} yet..."
+        end
+      end
+
+      # @!endgroup
+
+      REMOTE_PACKAGES_REQUIRED = ['make']
```

Comment:
For sets of requirements we've moved everything into options/presets.rb - so that we can track and view them all together in a single location.  I'm wary of having these set in the middle of a file where it would be difficult to monitor and can't be included in any options parsing.